fudge-ratio/src/main/webapp/index.html

```diff
@@ -2,11 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Fudge Ratio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```

Comment:
Are we going to end up using bootstrap (I mean I like it) but I see no code that uses this, so why not add it when we will use it?

Reply:
That was a line of code I copied from my portfolio code. I'll remove it. I don't know what we will be using for front end. We can discuss suggestions in a meeting or in the group chat?